src/args/arg_builder/positional.rs

```diff
@@ -130,11 +130,19 @@ impl<'n, 'e> PosBuilder<'n, 'e> {
 impl<'n, 'e> Display for PosBuilder<'n, 'e> {
     fn fmt(&self, f: &mut Formatter) -> Result {
         if self.settings.is_set(ArgSettings::Required) {
-            try!(write!(f, "<{}>", self.name));
+            if let Some(ref names) = self.val_names {
+                try!(write!(f, "{}", names.values().map(|n| format!("<{}>", n)).collect::<Vec<_>>().join(" ")));
```

Comment:
I am just curious, where does `.join()` come from? It seems like a very handy method, but I couldn't find it in `Vec` documentation...